internal/rbd/clone.go

```diff
@@ -52,31 +52,7 @@ func (rv *rbdVolume) checkCloneImage(ctx context.Context, parentVol *rbdVolume)
 	snap.RbdSnapName = rv.RbdImageName
 	snap.Pool = rv.Pool
 
-	// check if cloned image exists
-	err := rv.getImageInfo()
-	if err == nil {
-		// check if do we have temporary snapshot on temporary cloned image
-		sErr := tempClone.checkSnapExists(snap)
-		if sErr != nil {
-			if errors.Is(err, ErrSnapNotFound) {
-				return true, nil
-			}
-
-			return false, err
-		}
-		err = tempClone.deleteSnapshot(ctx, snap)
-		if err == nil {
-			return true, nil
-		}
-
-		return false, err
-	}
-	if !errors.Is(err, ErrImageNotFound) {
-		// return error if its not image not found
-		return false, err
-	}
-
-	err = tempClone.checkSnapExists(snap)
+	err := tempClone.checkSnapExists(snap)
```

Comment:
the commit `rbd: fix clone problem` does not contain any description. It would be good to describe what problem is fixed (and how).